app/models/job.rb

```diff
@@ -24,4 +24,8 @@ def assign_tech
   def self.by_tech(tech_name)
     joins(:technologies).where(technologies: {name: tech_name.downcase})
   end
+
+  def self.total_pages(num_of_items_per_page)
+     last_two_months.count / num_of_items_per_page
```

Comment:
This math isn't quite right.

Failing test case:

last_two_months.count == 30
num_of_items_per_page == 4

Currently returns 7, but should return 8.

7 * 4 = 28, not 30.